WordPress/Classes/ViewRelated/What's New/WPTabBarController+WhatIsNew.swift

```diff
@@ -0,0 +1,25 @@
+/// dependency container for the What's New / Feature Announcements scene
+extension WPTabBarController {
+
+    func makeWhatIsNewViewController() -> WhatIsNewViewController {
+        return WhatIsNewViewController(whatIsNewViewFactory: makeWhatIsNewView)
+    }
+
+    private func makeWhatIsNewView() -> WhatIsNewView {
+
+        var version = ""
+        if let versionNumber = Bundle.main.infoDictionary?["CFBundleShortVersionString"] as? String {
```

Comment:
We have a `NSBundle+VersionNumberHelper` that does this. Maybe you can use `Bundle.main.shortVersionString()`?

Reply:
yes absolutely, I had missed this helper, thank you!